src/entities/Entity.js

```diff
@@ -0,0 +1,10 @@
+class Entity {
```

Comment:
I don't think adding a class to be an entity in this moment adds a lot of benefits. Literally it only has one property. Also the type of that ID might change from entity to entity.

Reply:
I was planing on using it for the security stage to add 
- Created By
- Last Modified By
- Created Date
- Last modified Date
in a single object so that way I did not need to repeat these common properties on all entities. Do you still think I should remove this? 

About "Also the type of that ID might change from entity to entity."  : I was planning on using an ~8 digit id for all objects with different prefixes per object. Should I not? I'll research to try and understand reasons why it would change per entity and common practices on this. 

Feel free to share any insights you might have but do not feel obligated to, not that you have any reason to, I do not want to burden you with things I could research.